test/integration/crud/insert.test.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const { setupDatabase } = require('../shared');
+const { format: f } = require('util');
+const { expect } = require('chai');
+
+describe('crud - insert', function () {
+  before(function () {
+    return setupDatabase(this.configuration);
+  });
+
+  context('promise tests', () => {
+    it('Should correctly execute Collection.prototype.insertOne', {
+      metadata: {
+        requires: {
+          topology: ['single']
```

Comment:
why just single here?

Reply:
Same thoughts here as above.  I'll remove the topology restrictions here too